code/AssetLib/glTF2/glTF2AssetWriter.inl

```diff
@@ -550,7 +570,7 @@ namespace glTF2 {
 
     inline void Write(Value& obj, Mesh& m, AssetWriter& w)
     {
-		/****************** Primitives *******************/
+        /****************** Primitives *******************/
```

Comment:
Whitespace

Reply:
I actually fixed it, it was using tabs before while the rest of the file is white space

Reply:
That should go in a separate commit then. Mixing functional and whitespace changes makes reviewing harder and can create unnecessary conflicts.